smriprep/utils/tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pathlib import Path
+
+DERIV_SKELETON = Path(__file__).parent / ('derivatives.yml')
```

Comment:
Woops. Didn't click the green button on this one. Anyway, after all the effort purging `__file__`, I think we should not re-introduce it...

```suggestion
from niworkflows.data import Loader

load_data = Loader(__package__)

DERIV_SKELETON = load_data('derivatives.yml')
```